src/exceptions/PS-or-PDF-font-exception-20170817.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="PS-or-PDF-font-exception-20170817" name="PS/PDF font exception (2017-08-17)" listVersionAdded="3.2">
+      <crossRefs>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/master/LICENSE</crossRef>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/65962e27febc3883a17e651cdb23e783668c996f/LICENSE</crossRef>
```

Comment:
We don't have a policy on pinned vs. floating links, but there is some previous discussion [here][1].  As I said there, I prefer pinning the link when that's as easy as GitHub makes it.  But I don't think we need to have *both* pinned and floating links.  Folks can easily switch from the pinned link to a the floating version using GitHub's UI, without us having to link to the floating version directly.  I recommend keeping this line and dropping the previous `master` line.

[1]: https://github.com/spdx/license-list-XML/pull/512#discussion_r156771168